cancermuts/datasources.py

```diff
@@ -1161,6 +1165,159 @@ def _get_revel_from_gc(self, mutation, gc):
             self.log.debug(f"Downloaded revel score: {revel_scores}")
             return [ DbnsfpRevel(source=self, score=r) for r in revel_scores ]
 
+class RevelLocal(DynamicSource, object):
+    @logger_init
+    def __init__(self, database_dir, database_files=None):
+        description = "Local REVEL score annotation using Ensembl transcript ID"
+        super(RevelLocal, self).__init__(name='RevelLocal', version='1.0', description=description)
+
+        self._database_dir = database_dir
+
+        if database_files is None:
+            self._database_files = {
+                'revel': os.path.join(self._database_dir, 'revel_with_transcript_ids')
+            }
+        else:
+            self._database_files = database_files
```

Comment:
I think we can simplify this by just having one argument which is the the path to the REVEL score file, since we will always have just one